Signum.Engine.Extensions/Mailing/EmailTemplateRenderer.cs

```diff
@@ -29,7 +29,9 @@ public EmailMessageBuilder(EmailTemplateEntity template, Entity? entity, IEmailM
 
             this.queryName = QueryLogic.ToQueryName(template.Query.Key);
             this.qd = QueryLogic.Queries.QueryDescription(queryName);
-            this.smtpConfig = EmailTemplateLogic.GetSmtpConfiguration?.Invoke(template, (systemEmail?.UntypedEntity as Entity ?? entity)?.ToLite());
+            //this.smtpConfig = EmailTemplateLogic.GetSmtpConfiguration?.Invoke(template, null,(systemEmail?.UntypedEntity as Entity ?? entity));
+            this.smtpConfig = EmailTemplateLogic.GetSmtpConfiguration?.Invoke(template, (systemEmail?.UntypedEntity as Entity)?.ToLite(), null);

```

Comment:
ToLiteFat

Reply:
if SentException requires Sent date and Sent date it is not set because there is an error in the sendig before setting this date  setting SentException will give an error?

That is to say or I assure that Sent date is not null or I allow to establish Sent date with null in SentException